packages/babel-plugin/package.json

```diff
@@ -32,8 +32,8 @@
     "build": "tsup --config ../../tsup.config.ts",
     "typecheck": "tsc --noEmit --composite false",
     "test": "vitest run",
-    "lint": "eslint './src/**/*.{js,ts,jsx,tsx}'",
-    "lint:fix": "eslint --fix './src/**/*.{js,ts,jsx,tsx}'"
+    "lint": "eslint './src/**/*.{js,ts,jsx,tsx}' --report-unused-disable-directives --max-warnings 0",
```

Comment:
(note: I am interested in how we can share the same command across packages but not sure about clean way; shell script? 😅 )

Reply:
I don't know of such a method either...
There are cases where different commands need to be set for each package, so I think it's fine as it is.